WordPress/Classes/ViewRelated/Reader/Tab Navigation/DoubleBarControlConfigurer.swift

```diff
@@ -0,0 +1,36 @@
+import UIKit
+
+// Where to place this file?
```

Comment:
I am unfamiliar with the general folder structure so I placed it under `Tab Navigation` which is a helper adding for buttons to navigation bar. It is used from `NotificationsViewController` & `ReaderTabViewController`

Reply:
> Where to place this file?

Before I answer that, I'm a bit confused by the purpose of this file. I could be misunderstanding what this does, and a bit of a disclaimer - I've not been able to actually build and run this branch yet, so I've not been able to test it. So please forgive me if I got it wrong. But it seems to override the built-in navigation bar functionality of setting multiple right bar buttons (ex: `navigationItem.rightBarButtonItems = [searchBarButton, settingsButton]`) for the sole purpose of setting the buttons' width and height to 44. Could this not be done on each button before adding it to the nav bar? It seems to me sizing the buttons correctly is preferable to creating a whole custom stack view.

Reply:
No you're totally right. It only helped me avoid the code duplication there. The stack view grants us the control over the spacing between the bar button items. When I directly add `UIBarButtonItem` via `setBarButtonItems` function, there is a spacing between them that doesn't follow the current design. Another option I could think of was to use the appearance API to set the spacing to 0.
`UIStackView.appearance(whenContainedInInstancesOf: [UINavigationBar.self]).spacing`
I thought this is a bit too intrusive for me to use without communicating with the team. 

Another option I've found on SO is to use a custom spacer view in between:
`let space = UIBarButtonItem(barButtonSystemItem: .fixedSpace...`
I did not know about the `fixedSpace` API which might save us from all the logic in this file. I quite like this approach. If you agree and you don't have another specific one in mind, I would like to try it out and remove that file if it works.

Thanks for the feedback, I have found out about the latter API thanks to it!

Reply:
I feel like we're getting sidetracked by an an issue that is unrelated to this project and isn't necessary to fix with it. A few minor sizing/spacing oddities clearly isn't causing any issues (and no one has noticed 😄 ).

 My suggestion is this:
- Let's not worry about fixing these minor nav bar spacing/sizing issues right now, and revert to using the "normal" way of adding nav bar buttons (`navigationItem.rightBarButtonItems`).
- If you don't mind, please create a GitHub issue in the WP repo for the Reader nav bar issues, describing the problem and your research/findings. The screenshots you created for Matt Miklic would be helpful too.
- When this project is finished, we can add information to that issue for the Notifications nav bar (if the issue is still the same).